android/build.gradle

```diff
@@ -31,59 +20,34 @@ def safeExtGet(prop, fallback) {
   rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-def DEFAULT_COMPILE_SDK_VERSION = 28
-def DEFAULT_BUILD_TOOLS_VERSION = "28.0.2"
-def DEFAULT_MIN_SDK_VERSION = 19
-def DEFAULT_TARGET_SDK_VERSION = 27
-
 android {
-  compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
-  buildToolsVersion safeExtGet('buildToolsVersion', DEFAULT_BUILD_TOOLS_VERSION)
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
-  }
-  kotlinOptions {
-    jvmTarget = JavaVersion.VERSION_1_8
-  }
+  compileSdkVersion safeExtGet('compileSdkVersion', 32)
   defaultConfig {
-    minSdkVersion safeExtGet('minSdkVersion', DEFAULT_MIN_SDK_VERSION)
-    targetSdkVersion safeExtGet('targetSdkVersion', DEFAULT_TARGET_SDK_VERSION)
-    versionCode 1
-    versionName "1.0"
+    minSdkVersion safeExtGet('minSdkVersion', 23)
+    targetSdkVersion safeExtGet('targetSdkVersion', 32)
   }
   lintOptions {
     abortOnError false
   }
 }
 
 repositories {
-  mavenLocal()
-  google()
-  jcenter()
-    // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
   maven {
-    url("../node_modules/react-native/android")
+    url('../node_modules/react-native/android')
   }
+  google()
+  mavenCentral()
 }
 
 dependencies {
-    // React-Native
-    implementation 'com.facebook.react:react-native:+'
-
-    // Zcash
-    implementation  "cash.z.ecc.android:zcash-android-sdk:${versions.zcash}"
-
-    // Process Room annotations
-    kapt "androidx.room:room-compiler:${versions.room}"
+  implementation 'com.facebook.react:react-native:+'
 
-    implementation "androidx.paging:paging-runtime-ktx:2.1.2"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9"
+  // Compiler plugin for Room annotation support:
+  kapt 'androidx.room:room-compiler:2.3.0'
 
-    implementation ("androidx.appcompat:appcompat:1.3.1") {
-        version {
-            strictly '1.3.1'
-        }
-    }
+  implementation 'androidx.appcompat:appcompat:1.4.1'
```

Comment:
This won't build with 1.4.1 for sure but I think the syntax means it will use a newer version if found. I don't know if this build with a newer version.

Reply:
I tested it on develop with these changes, and it’s fine once I knock out the root problem in edge-login-ui-rn.